tests/EnvMapperTest.php

```diff
@@ -37,6 +41,10 @@ public function mapping_different_types_with_defaults_parses_correctly(): void
         self::assertNotNull($env->PATH);
         self::assertNotNull($env->hostname);
         self::assertNotNull($env->shlvl);
+        self::assertSame('01234', $env->zipCode);
+        self::assertSame(true, $env->bool);
+        self::assertEquals(StringBackedEnum::FOO, $env->stringBackedEnum);
+        self::assertEquals(IntegerBackedEnum::BAR, $env->integerBackedEnum);
```

Comment:
I think these were missed.

Reply:
Whoops, that'll teach me no to use refactor>rename... Fixed.